index.js

```diff
@@ -1,20 +1,18 @@
 const express = require('express')
 const bodyParser = require('body-parser')
 const MongoClient = require('mongodb').MongoClient
-//what does 'this'v do?
 const app = express()
 const port = 4400
 const url = 'mongodb://localhost:27017'
 var jsonParser = bodyParser.json()
 
-
 var addNewAccount = (db, accountData) => {
     var collection = db.collection('accounts')
     console.log(accountData)
```

Comment:
```suggestion

```
Make sure to get rid of the console.log here as those are just for testing

Reply:
Removed :)